script/demoScripts/demoMayan.ts

```diff
@@ -20,83 +20,86 @@ const main = async () => {
 
   const provider = new ethers.providers.JsonRpcProvider(RPC_URL)
   const signer = new ethers.Wallet(PRIVATE_KEY as string, provider)
-  const mayan = MayanBridgeFacet__factory.connect(LIFI_ADDRESS, provider)
-  const deadline = Math.floor(Date.now() / 1000) + 60 * 10 // 10 minutes from the current Unix time
+  const mayan = MayanFacet__factory.connect(LIFI_ADDRESS, provider)
 
   const address = await signer.getAddress()
 
   let tx
 
-  const quote: Quote = await fetchQuote({
-    amount: 10,
-    fromToken: BSC_USDT_ADDRESS,
-    toToken: '0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174',
-    fromChain: 'bsc',
+  const quote: Quote[] = await fetchQuote({
+    amount: 1,
+    fromToken: constants.AddressZero,
+    toToken: POLYGON_USDT_ADDRESS,
+    fromChain: 'ethereum',
     toChain: 'polygon',
-    slippage: 3,
+    slippageBps: 300,
   })
 
-  const payload = await getSwapFromEvmTxPayload(
-    quote,
+  const payload = getSwapFromEvmTxPayload(
+    quote[0],
+    address,
     address,
-    deadline,
     null,
     address,
-    56,
-    provider
+    1,
+    null,
+    null
   )
 
-  const iface = IMayanBridge__factory.createInterface()
-  const parsed = iface.parseTransaction({ data: payload.data as string })
-
-  const token = ERC20__factory.connect(BSC_USDT_ADDRESS, provider)
-
-  const bridgeData: ILiFi.BridgeDataStruct = {
-    transactionId: utils.randomBytes(32),
-    bridge: 'Mayan',
-    integrator: 'ACME Devs',
-    referrer: '0x0000000000000000000000000000000000000000',
-    sendingAssetId: BSC_USDT_ADDRESS,
-    receiver: address,
-    minAmount: utils.parseEther('10'),
-    destinationChainId: 137,
-    hasSourceSwaps: false,
-    hasDestinationCall: false,
-  }
-
-  const mayanData: MayanBridgeFacet.MayanBridgeDataStruct = {
-    mayanAddr: parsed.args.recipient.mayanAddr,
-    referrer: utils.hexZeroPad('0x', 32),
-    tokenOutAddr: parsed.args.tokenOutAddr,
-    receiver: parsed.args.recipient.destAddr,
-    swapFee: parsed.args.relayerFees.swapFee,
-    redeemFee: parsed.args.relayerFees.redeemFee,
-    refundFee: parsed.args.relayerFees.refundFee,
-    transferDeadline: parsed.args.criteria.transferDeadline,
-    swapDeadline: parsed.args.criteria.swapDeadline,
-    amountOutMin: parsed.args.criteria.amountOutMin,
-    unwrap: parsed.args.criteria.unwrap,
-    gasDrop: parsed.args.criteria.gasDrop,
-  }
+  console.log('payload', payload)
 
-  console.info('Dev Wallet Address: ', address)
-  console.info('Approving USDT...')
-  const gasPrice = await provider.getGasPrice()
-  tx = await token
-    .connect(signer)
-    .approve(LIFI_ADDRESS, constants.MaxUint256, { gasPrice })
-  await tx.wait()
-  console.info('Approved USDT')
-  console.info('Bridging USDT...')
-  tx = await mayan
-    .connect(signer)
-    .startBridgeTokensViaMayanBridge(bridgeData, mayanData, { gasPrice })
-  await tx.wait()
-  console.info('Bridged USDT')
+  // const iface = IMayanBridge__factory.createInterface()
```

Comment:
Intentional to have this commented code here?

Reply:
Whoops. Updated.